nuxeo-core/nuxeo-core-management/src/main/java/org/nuxeo/ecm/core/management/statuses/AdministrativeStatusManagerImpl.java

```diff
@@ -107,6 +102,13 @@ public void onNuxeoServerStartup() {
 
         doNotifyAllStatuses();
 
+        scheduler = Executors.newScheduledThreadPool(1, new ThreadFactory() {
+            @Override
+            public Thread newThread(Runnable r) {
+                return new Thread(r, "Nuxeo-Administrative-Statuses-Notify-Scheduler");
+            }
+        });
```

Comment:
We should use our standard scheduler service for that, but that's for another NXP.